openreview/api/client.py

```diff
@@ -1391,6 +1391,32 @@ def get_group_edit(self, id):
         n = response.json()['edits'][0]
         return Edit.from_json(n)
 
+    def get_group_edits(self, group_id = None, invitation = None, with_count = False, sort = None, trash = None):
+        """
+        Gets a list of edits for a group. The edits that will be returned match all the criteria passed in the parameters.
+
+        :return: List of edits
+        :rtype: list[Edit]
+        """
+        params = {}
+        if group_id:
+            params['group.id'] = group_id
+        if invitation:
+            params['invitation'] = invitation
+        if sort:
+            params['sort'] = sort
+        params['trash'] = 'true' if trash == True else 'false'
```

Comment:
I thought that `tools.format_params` already did that for us.

Reply:
Yes, you're right. I've fixed this in both `get_group_edits()` and `get_note_edits()`